### ISSUE TITLE
Monkey.Robotics component missing in components list

### ISSUE THREAD

**Issue by nikitavarat** [xamarin/Monkey.Robotics#64]
To add component in project when I tried to search with keyword 'Robotics', got message 'No components found'. It seems this component has been removed from component list.

**Comment by TWhidden** [comment]
Yes, same issue for me. Trying to do an Bluetooth LE project.  Was trying to use this as a sample with XF.

**Comment by himion0** [comment]
@TWhidden Did you find something else to replace it?

**Comment by AmperayaniMadan** [comment]
Is there a way to use this source code to generate the component, or import this code into our project ??

**Comment by AmperayaniMadan** [comment]
I have found two other plugins based on Monkey.Robotics components.
1. [link to plugin] (https://www.nuget.org/packages/XamarinBluetoothLE)
    - [link to its GitHub page] (https://github.com/tbrushwyler/Xamarin.BluetoothLE/)
2. [link to xaber plugin] (https://github.com/xabre/xamarin-bluetooth-le)

I am using the first one, found it more suitable for my requirements.
